templates/tools/dockerfile/python_debian11.include

```diff
@@ -0,0 +1,53 @@
+FROM debian:bullseye
+  
+# Install Git and basic packages.
+RUN apt-get update && apt-get install -y ${'\\'}
+  autoconf ${'\\'}
+  autotools-dev ${'\\'}
+  build-essential ${'\\'}
+  bzip2 ${'\\'}
+  ccache ${'\\'}
+  curl ${'\\'}
+  dnsutils ${'\\'}
+  gcc ${'\\'}
+  gcc-multilib ${'\\'}
+  git ${'\\'}
+  golang ${'\\'}
+  gyp ${'\\'}
+  lcov ${'\\'}
+  libc6 ${'\\'}
+  libc6-dbg ${'\\'}
+  libc6-dev ${'\\'}
+  libgtest-dev ${'\\'}
+  libtool ${'\\'}
+  make ${'\\'}
+  perl ${'\\'}
+  strace ${'\\'}
+  telnet ${'\\'}
+  unzip ${'\\'}
+  wget ${'\\'}
+  zip && apt-get clean
+
+#================
+# Build profiling
+RUN apt-get update && apt-get install -y time && apt-get clean
+
+# Install Python 3.7 from source (and installed as a default python3)
+# (Bullseye comes with Python 3.9 which isn't supported by pytype yet)
+RUN apt update && apt install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev ${'\\'}
```

Comment:
why exactly do we need python  in the sanity docker image  (and why do we need a version that's supported by pytype?)

while building from source is fine, it make the docker image rebuild time much longer and it also feels a bit non-standard, so I'd like to avoid it whenever we can. 

is there an ETA for pytype going to start supporting python3.9? Can you at least add a todo to get rid of python3.7 once the blocker is gone?

Reply:
Sanity docker needs Python because a couple of python linters will run (e.g. pytype) and pytype doesn't support Python 3.9 yet and there is not plan yet (https://github.com/google/pytype/issues/749) so I don't expect that they support anytime soon.

I don't like to build something from source either since it increases the build time and not easy to maintain but python isn't that bad for both. It just needs 1-2 minutes more and build script isn't that cryptic.

Reply:
Once pytypes supports python 3.9, we can just use python3.9 package as it.

Reply:
Fair enough.